index.js

```diff
@@ -145,10 +145,18 @@ function createThreadInterceptor (opts) {
   return res
 }
 
-function wire (server, port, opts) {
+function wire (newServer, port, opts) {
```

Comment:
Can you please make this an option object instead?

```js
wire({
  server,
  port,
  ...undiciOpts
})
```

Reply:
Sure, on my way.

Reply:
Done!